repose-aggregator/extensions/pom.xml

```diff
@@ -18,9 +18,33 @@
         No other filters should be developed here.
     </description>
 
+    <properties>
+        <api-checker.version>1.1.3</api-checker.version>
```

Comment:
I was following the format that API-Checker uses:
* https://github.com/rackerlabs/api-checker/blob/master/pom.xml#L53

What would you rather it be?
All dashes or all dots?